skycultures/bali/index.json

```diff
@@ -0,0 +1,859 @@
+    },
+    {
+      "id": "CON bali 016",
+      "lines": [[
+        64962,
+        56343,
+        52943,
+        51069,
+        49841,
+        48356,
+        46390,
+        43813,
+        43109
+      ]],
+	  "image": {
+		  "file": "illustrations/Naga.png",
+		  "size": [1408, 809],
+		  "anchors": [
+		  {"pos": [147, 294], "hip": 64962},
+		  {"pos": [466, 567], "hip": 56343},
+		  {"pos": [1152, 274], "hip": 43813}]
+	  },
+      "common_name": {
+        "english": "a dragon",
+        "native": "ᬦᬕ",
+        "pronounce": "naga",
+        "IPA": "/ˈna.ɡa/"
+      }
+    },
+    {
+      "id": "CON bali 017",
+      "lines": [[
+        92855,
+        93506
+      ]],
+	  "image": {
+		  "file": "illustrations/Prahu_Pegat.png",
+		  "size": [1227, 600],
+		  "anchors": [
+		  {"pos": [498, 333], "hip": 92855},
+		  {"pos": [679, 328], "hip": 93506},
+		  {"pos": [603, 413], "hip": 93864}]
+	  },
+      "common_name": {
+        "english": "a broken ship",
+        "native": "ᬧ᭄ᬭᬳᬸᬧᭂᬕᬢ᭄",
+        "pronounce": "prahu pěgat",
+        "IPA": "/pra.hu.pə.gat/"
+      }
+    },
+    {
+      "id": "CON bali 018",
+      "lines": [[
+        107556,
+        106985,
+        100064,
+        100345,
+        102485,
+        102978,
+        105881,
+        107556
+      ]],
+	  "image": {
+		  "file": "illustrations/Udang.png",
+		  "size": [1200, 714],
+		  "anchors": [
+		  {"pos": [627, 632], "hip": 102978},
+		  {"pos": [958, 300], "hip": 100064},
+		  {"pos": [259, 204], "hip": 107556}]
+	  },
+      "common_name": {
+        "english": "a prawn",
+        "native": "ᬳᬸᬤᬂ",
+        "pronounce": "udang",
+        "IPA": "/ˈudaŋ/"
+      }
+    },
+    {
+      "id": "CON bali 019",
+      "lines": [[
+        6193,
+        5742,
+        5571,
+        7097,
+        8198,
+        9487,
+        8833,
+        7884,
+        7007,
+        4906,
+        3786,
+        116771,
+        115830,
+        114971,
+        115738,
+        116928
+      ]],
+	  "image": {
+		  "file": "illustrations/Puwuh_Atarung.png",
+		  "size": [1200, 1077],
+		  "anchors": [
+		  {"pos": [690, 313], "hip": 9487},
+		  {"pos": [113, 692], "hip": 114971},
+		  {"pos": [841, 729], "hip": 5571}]
+	  },
+      "common_name": {
+        "english": "battling quails",
+        "native": "ᬧᬸᬯᬸᬄᬳᬢᬭᬸᬂ",
+        "pronounce": "puwuh atarung",
+        "IPA": "/ˈpu.wuh.a.ta.ruŋ/"
+      }
+    },
+    {
+      "id": "CON bali 020",
+      "lines": [[
+        67301,
+        65378,
+        62956,
+        59774,
+        58001
+      ]],
+	  "image": {
+		  "file": "illustrations/Larung_Paglangan.png",
+		  "size": [1500, 1150],
+		  "anchors": [
+		  {"pos": [195, 811], "hip": 58001},
+		  {"pos": [1213, 802], "hip": 67301},
+		  {"pos": [659, 1001], "hip": 62956}]
+	  },
+      "common_name": {
+        "english": "throwing ashes",
+        "native": "ᬮᬭᬸᬂᬧᬕ᭄ᬮᬗᬦ᭄",
+        "pronounce": "larung paglangan",
+        "IPA": "/la.ruŋ.pag.la.ŋ'an/"
```

Comment:
What does the apostrophe signify here? If it's supposed to be a stress mark, then it's denoted by `ˈ` and placed before the stressed syllable. If it's a glottal stop, then the proper symbol is `ʔ`.